platforms/android/build.gradle

```diff
@@ -193,26 +193,6 @@ android {
     buildToolsVersion "23.0.2"
 
     if (Boolean.valueOf(cdvBuildMultipleApks)) {
```

Comment:
why keep this `if` when its `then` clause is now empty?

Reply:
This file shouldn't even be in source control at all, but we can't remove it until we move all the custom changes to another folder. The end goal is to delete `platforms` entirely.